middleman-cli/lib/middleman-cli/templates/extension/gemspec

```diff
@@ -11,8 +11,7 @@ Gem::Specification.new do |s|
   # s.summary     = %q{A short summary of your extension}
   # s.description = %q{A longer description of your extension}
 
-  s.files         = `git ls-files`.split("\n")
-  s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
+  s.files       = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(fixtures|features|spec)/}) }
```

Comment:
It wouldn't work if gems are built from a parent folder. Not sure, if that's ok.

Reply:
We build gems with the `bundle exec rake publish` command from the root.

Reply:
@tdreyno thanks! Just tested it, works correctly